src/keysManager.js

```diff
@@ -16,7 +16,13 @@ export default class KeysManager {
 
   async isInitialKeyValid(initialKey) {
     return new Promise((resolve, reject) => {
-      this.keysInstance.methods.initialKeys(initialKey).call().then(function(result){
+      let getInitialKeyStatus
+      if (this.keysInstance.methods.getInitialKeyStatus) {
```

Comment:
Let's define `const methods = this.keysInstance.methods` and reuse it.

Reply:
@vbaranov Done in https://github.com/poanetwork/poa-dapps-keys-generation/pull/75/commits/fe8457661d61809953f43bd36b82ca570cf0a9de.